workers/unity/Assets/Gdk/Core/Tests/Editmode/Components/OptionTests.cs

```diff
@@ -0,0 +1,179 @@
+using System;
+using NUnit.Framework;
+
+namespace Improbable.Gdk.Core.EditmodeTests
+{
+    [TestFixture]
+    public class OptionTests
+    {
+        [Test]
+        public void Parameterless_constructor_creates_empty_option()
+        {
+            var option = new Option<bool>();
+            Assert.AreEqual(false, (bool)option.HasValue);
+        }
+
+        [Test]
+        public void Constructor_with_parameters_creates_option_with_payload()
+        {
+            var payload = true;
+            var option = new Option<bool>(payload);
+            Assert.AreEqual(true, (bool)option.HasValue);
+            Assert.AreEqual(payload, option.Value);
```

Comment:
You repeat this same assertion below

Reply:
Put this in a one time setup block and assert on it there. Then they other tests can assume this for setup

Reply:
Removed redundant assertions.